src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2200,6 +2213,23 @@ Object toStyleConstants(StyleConstants key, View v) {
             return Integer.valueOf(getValue(null, null));
         }
 
+        @Override
+        public int hashCode() {
+            int hashCode = Float.hashCode(value) | Boolean.hashCode(index);
+            if (lu != null) {
+                hashCode = lu.hashCode();
+            }
+            return hashCode;
+        }
```

Comment:
To be consistent with `LengthValue` and `LengthUnit`:
```suggestion
        public int hashCode() {
            return Float.hashCode(value)
                   | Boolean.hashCode(index)
                   | Objects.hashCode(lu);
        }
```